cosinnus_message/rocket_chat.py

```diff
@@ -728,25 +730,36 @@ def users_update(self, user, request=None, force_user_update=False, update_passw
         profile = user.cosinnus_profile
         rocket_email = user_data.get('emails', [{}])[0].get('address', None)
         # rocket_mail_verified = user_data.get('emails', [{}])[0].get('verified', None)
-        rocket_username = self._get_unique_username(profile)
-        if not rocket_username:
-            return
-        if force_user_update or user_data.get('name') != rocket_username or rocket_email != profile.rocket_user_email:
+        rocket_username = user_data.get('username')
```

Comment:
2. Here we move from `user_data.get('name')` to `user_data.get('username')`. Is this backwards compatible with RC 6.x or do we need to time this change exactly with the swap to RC 7.x?

Reply:
This is already incorrect with RC 6. `name` is the full user name (`profile.get_external_full_name()`) and comparing it to username lead to always considering that the username has changed.